src/config/risingwave.toml

```diff
@@ -6,6 +6,7 @@ heartbeat_interval_ms = 1000
 [streaming]
 checkpoint_interval_ms = 250
 in_flight_barrier_nums = 40
+worker_node_parallelism = 4
```

Comment:
Ideally, we should use different config files for CI, playground or production. Let's do it in the future.